src/app/services/po-blog-data.service.ts

```diff
@@ -0,0 +1,17 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Blog } from '../domain/blog.model';
+import { Observable } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class PoBlogDataService {
```

Comment:
Services need not have Po prefix. Only components need to have them as they are exposed to the DOM.